ports/spatial-hash/portfile.cmake

```
@@ -0,0 +1,26 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO MIT-SPARK/Spatial-Hash
+    REF bf592f26d84beca96e3ddc295ee1cf5b7341dee5
+    SHA512 c6e0c0475f2ca9bd9b21b227874202a12191496a446e44c493d6a181636912a342c56a8742cb5597a164f108bce74ec9534e224db4fa916c76930b232c82895f
+    PATCHES
+    compatible_vcpkg_cmake.patch
+        fix_cmake_config.patch
+)
+
+set(VCPKG_POLICY_SKIP_COPYRIGHT_CHECK enabled) # Since there is a '-' character in the port name it copies  to wrong directory (shared/spatial-hash), so we do it manually.
+
+vcpkg_configure_cmake(
```

```suggestion
vcpkg_cmake_configure(
```
You need use the new function: `vcpkg_cmake_configure`.